test/com/dataengi/crm/identities/daos/CompaniesDAOSpec.scala

```diff
@@ -42,6 +98,27 @@ class CompaniesDAOSpec extends PlaySpecification with AuthenticationContext with
       getByNameResult.isRight === true
     }
 
+    "Add list of companies" in {
+      val company1         = Company(name = "Test")
+      val company2         = Company(name = "Test1")
+      val CompanyList      = List(company1, company2)
+      val addCompanyResult = companiesDAO.add(CompanyList).await()
+      println(s"[companies-dao][add] $addCompanyResult}")
+      addCompanyResult.isRight === true
+
+      val id1            = addCompanyResult.right.get.head
+      val id2            = addCompanyResult.right.get.last
+      val getByCompanyId = companiesDAO.get(id1).await()
+      getByCompanyId.isRight === true
+      val getByIdCompany = companiesDAO.get(id2).await()
+      getByIdCompany.isRight === true
+    }
+
+    "list of companies" in {
+      val getByIdCompany = companiesDAO.all.await()
+      getByIdCompany.isRight === true
```

Comment:
maybe test steps must be
- insert few companies into empty storage
- call all
- check elements of list